merge-process.md

```diff
@@ -12,6 +12,10 @@ Acceptance of a PR is accomplished as follows:
   - PR contains only documentation changes
     - two assenting votes (approvals) have been cast with no dissenting votes
       (requests for changes).
+  - PR contains an urgent hotfix that is required to fix a critical regression. This is to
+    be used only in exceptional cases
+    - two assenting votes (approvals) have been cast with no dissenting votes
+      (requests for changes).   
   - PR contains any code changes
     - five assenting votes (approvals) have been cast with no dissenting votes
       (requests for changes).
```

Comment:
```suggestion
      (requests for changes).
- if all PR aproval-group members assent, the three busines days waiting periode does no apply.
```

as suggested in the discussion

Reply:
like so maybe?
"- if all PR aproval-group members assent, the three busines days waiting periode does no apply, except for hotfixes".

Reply:
no this makes no sense, sorry.

the exception needs to be put into the hotfix scenario

Reply:
```
  - PR contains only documentation changes
    - two assenting votes (approvals) have been cast with no dissenting votes
      (requests for changes).
  - PR contains an urgent hotfix that is required to fix a critical regression. This is to
    be used only in exceptional cases.
    - two assenting votes (approvals) have been cast with no dissenting votes
      (requests for changes).   
    -  The three-working-days rule does not apply in this case.
  - PR contains any code changes
    - five assenting votes (approvals) have been cast with no dissenting votes
      (requests for changes).
    -  The three-working-days rule does not apply in case all PR approval members have assented.

```

Reply:
This would basically mean that the 3-days-rule applies only to documentation changes.